sourced/ml/tests/test_merge_coocc_entry.py

```diff
@@ -0,0 +1,64 @@
+import os
+import shutil
+import tempfile
+import unittest
+
+import numpy as np
+
+from sourced.ml.cmd import merge_coocc_entry
+from sourced.ml.models import Cooccurrences
+from sourced.ml.tests import models
+
+
+def get_args(_input_dir, _no_spark):
+    class args:
+        input = [os.path.join(_input_dir, x) for x in os.listdir(_input_dir)]
+        output = os.path.join(_input_dir, "res_coocc.asdf")
+        docfreq = models.COOCC_DF
+        pause = False
+        filter = "**/*.asdf"
+        log_level = "INFO"
+        no_spark = _no_spark
+    return args
+
+
+class MergeCooccEntry(unittest.TestCase):
```

Comment:
Is there a test which checks the saturation?

Reply:
No, will add